testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -150,37 +150,37 @@ def tearDown(self):
         del self.at4
 
     def test_lt(self):
-        assert_((self.at1 < self.at2) == True)
-        assert_((self.at3 < self.at2) == False)
-        assert_((self.at2 < self.at2) == False)
+        assert_(self.at1 < self.at2)
+        assert_(self.at3 >= self.at2)
```

Comment:
The test is about the `<` operator. So that operator should be used. Maybe `assert_(not (self.at3 < self.at2))`?